diffeqzoo/ivps.py

```diff
@@ -874,3 +874,61 @@ def oregonator(
         initial_values=initial_values,
         time_span=time_span,
     )
+
+
+def goodwin(
+    *,
+    initial_values=(
+        0.0,
+        0.0,
+    ),
+    time_span=(0.0, 25.0),
+    parameters=(
+        10,
+        1.0,
+        3.0,
+        0.5,
+        1.0,
+    ),
+):
+    r"""Construct the Goodwin Oscillator dynamics.
+
+    Describes a mechanism for periodic protein expression described by Goodwin (1965).
+    The first dimension describes the mRNA concentration,
+    the last dimension the protein inhibiting mRNA production,
+    and the remaining dimensions correspond to intermediate protein species.
+    r > 8 leads to oscillatory behavior.
+    It is a n-dimensional ODE initial value problem.
+    The length of the parameters needs to be `len(initial_values)+3`.
+
+    Common problem for parameter inference, where the posterior has a multimodal distribution.
+
+    .. math::
+        \dot u(t) = f(u(t))
+
+    .. collapse:: BibTex for Goodwin (1965)
+
+        .. code-block:: tex
+            @article{goodwin1965oscillatory,
```

Comment:
Could a lack of an empty line be the reason the docs fail?

```suggestion

            @article{goodwin1965oscillatory,
```